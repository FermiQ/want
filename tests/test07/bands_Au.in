
 &INPUT
   prefix = 'gold'
   postfix = '_WanT'
!   fileout = 'gold_WanT_bands_NN.dat'
!   use_nn(3) = .TRUE.
   work_dir = './SCRATCH/'
   nkpts_in  = 2
   nkpts_max = 100
 /
G   0.0 0.0 0.0
Z   0.0 0.0 0.5
