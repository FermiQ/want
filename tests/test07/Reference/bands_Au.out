  ======================================================================
              =                                            =
              =     *** WanT *** Wannier Transport Code    =
              =        (www.wannier-transport.org)         =
              =      Ultra Soft Pseudopotential Implem.    =
              =                                            =
  ======================================================================


  Program <bands>  v. 1.1.1  starts ...
  Date 12Jul2005 at 15:21:26 


  PW-DFT data read from file: gold.export/index.xml
  Space data read from file: gold_WanT.space
  Hamiltonian data read from file: gold_WanT.ham

  ======================================================================
  =                              DFT data                              =
  ======================================================================

<LATTICE>
  Alat  =      20.0000000 (Bohr)
  Alat  =      10.5835400 (Ang )
  Omega =    3200.0000000 (Bohr^3)
  Omega =     474.1905097 (Ang^3 )

  Crystal axes:
                in units of Bohr                 in Alat units
    a(1) = ( 20.0000  0.0000  0.0000 )     (   1.0000  0.0000  0.0000 )
    a(2) = (  0.0000 20.0000  0.0000 )     (   0.0000  1.0000  0.0000 )
    a(3) = (  0.0000  0.0000  8.0000 )     (   0.0000  0.0000  0.4000 )
  Crystal axes: (Ang)
    a(1) = ( 10.5835  0.0000  0.0000 )
    a(2) = (  0.0000 10.5835  0.0000 )
    a(3) = (  0.0000  0.0000  4.2334 )

   Reciprocal lattice vectors:
                in units of Bohr^-1              in 2Pi/Alat units
    b(1) = (  0.3142  0.0000  0.0000 )     (   1.0000  0.0000  0.0000 )
    b(2) = (  0.0000  0.3142  0.0000 )     (   0.0000  1.0000  0.0000 )
    b(3) = (  0.0000  0.0000  0.7854 )     (   0.0000  0.0000  2.5000 )
</LATTICE>

<K-POINTS>
  nkpts =    8
  Monkhorst-Pack grid:      nk = (  1  1  8 ),      shift = (  0  0  0 )
  K-point calculation: (cart. coord. in Bohr^-1)
    k point   1:   (   0.00000  0.00000  0.00000 ),   weight =   0.1250
    k point   2:   (   0.00000  0.00000  0.09817 ),   weight =   0.1250
    k point   3:   (   0.00000  0.00000  0.19635 ),   weight =   0.1250
    k point   4:   (   0.00000  0.00000  0.29452 ),   weight =   0.1250
    k point   5:   (   0.00000  0.00000  0.39270 ),   weight =   0.1250
    k point   6:   (   0.00000  0.00000  0.49087 ),   weight =   0.1250
    k point   7:   (   0.00000  0.00000  0.58905 ),   weight =   0.1250
    k point   8:   (   0.00000  0.00000  0.68722 ),   weight =   0.1250
</K-POINTS>


  ======================================================================
  =                     Band interpolation by WFs                      =
  ======================================================================


  Generating kpts:   2      Segments:   1
  Total kpts number: 100    Max kpt number: 100


  Generating kpts [cart. coord. Bohr^-1]
      k point   1:   (   0.00000  0.00000  0.00000 )    G 
      k point   2:   (   0.00000  0.00000  0.39270 )    Z 

  Number of kpts in each segment
      line   1:     100


  Generated kpts  [cart. coord. Bohr^-1]
      k point   1:   (   0.00000  0.00000  0.00000 ) 
      k point   2:   (   0.00000  0.00000  0.00393 ) 
      k point   3:   (   0.00000  0.00000  0.00785 ) 
      k point   4:   (   0.00000  0.00000  0.01178 ) 
      k point   5:   (   0.00000  0.00000  0.01571 ) 
      k point   6:   (   0.00000  0.00000  0.01963 ) 
      k point   7:   (   0.00000  0.00000  0.02356 ) 
      k point   8:   (   0.00000  0.00000  0.02749 ) 
      k point   9:   (   0.00000  0.00000  0.03142 ) 
      k point  10:   (   0.00000  0.00000  0.03534 ) 
      k point  11:   (   0.00000  0.00000  0.03927 ) 
      k point  12:   (   0.00000  0.00000  0.04320 ) 
      k point  13:   (   0.00000  0.00000  0.04712 ) 
      k point  14:   (   0.00000  0.00000  0.05105 ) 
      k point  15:   (   0.00000  0.00000  0.05498 ) 
      k point  16:   (   0.00000  0.00000  0.05890 ) 
      k point  17:   (   0.00000  0.00000  0.06283 ) 
      k point  18:   (   0.00000  0.00000  0.06676 ) 
      k point  19:   (   0.00000  0.00000  0.07069 ) 
      k point  20:   (   0.00000  0.00000  0.07461 ) 
      k point  21:   (   0.00000  0.00000  0.07854 ) 
      k point  22:   (   0.00000  0.00000  0.08247 ) 
      k point  23:   (   0.00000  0.00000  0.08639 ) 
      k point  24:   (   0.00000  0.00000  0.09032 ) 
      k point  25:   (   0.00000  0.00000  0.09425 ) 
      k point  26:   (   0.00000  0.00000  0.09817 ) 
      k point  27:   (   0.00000  0.00000  0.10210 ) 
      k point  28:   (   0.00000  0.00000  0.10603 ) 
      k point  29:   (   0.00000  0.00000  0.10996 ) 
      k point  30:   (   0.00000  0.00000  0.11388 ) 
      k point  31:   (   0.00000  0.00000  0.11781 ) 
      k point  32:   (   0.00000  0.00000  0.12174 ) 
      k point  33:   (   0.00000  0.00000  0.12566 ) 
      k point  34:   (   0.00000  0.00000  0.12959 ) 
      k point  35:   (   0.00000  0.00000  0.13352 ) 
      k point  36:   (   0.00000  0.00000  0.13744 ) 
      k point  37:   (   0.00000  0.00000  0.14137 ) 
      k point  38:   (   0.00000  0.00000  0.14530 ) 
      k point  39:   (   0.00000  0.00000  0.14923 ) 
      k point  40:   (   0.00000  0.00000  0.15315 ) 
      k point  41:   (   0.00000  0.00000  0.15708 ) 
      k point  42:   (   0.00000  0.00000  0.16101 ) 
      k point  43:   (   0.00000  0.00000  0.16493 ) 
      k point  44:   (   0.00000  0.00000  0.16886 ) 
      k point  45:   (   0.00000  0.00000  0.17279 ) 
      k point  46:   (   0.00000  0.00000  0.17671 ) 
      k point  47:   (   0.00000  0.00000  0.18064 ) 
      k point  48:   (   0.00000  0.00000  0.18457 ) 
      k point  49:   (   0.00000  0.00000  0.18850 ) 
      k point  50:   (   0.00000  0.00000  0.19242 ) 
      k point  51:   (   0.00000  0.00000  0.19635 ) 
      k point  52:   (   0.00000  0.00000  0.20028 ) 
      k point  53:   (   0.00000  0.00000  0.20420 ) 
      k point  54:   (   0.00000  0.00000  0.20813 ) 
      k point  55:   (   0.00000  0.00000  0.21206 ) 
      k point  56:   (   0.00000  0.00000  0.21598 ) 
      k point  57:   (   0.00000  0.00000  0.21991 ) 
      k point  58:   (   0.00000  0.00000  0.22384 ) 
      k point  59:   (   0.00000  0.00000  0.22777 ) 
      k point  60:   (   0.00000  0.00000  0.23169 ) 
      k point  61:   (   0.00000  0.00000  0.23562 ) 
      k point  62:   (   0.00000  0.00000  0.23955 ) 
      k point  63:   (   0.00000  0.00000  0.24347 ) 
      k point  64:   (   0.00000  0.00000  0.24740 ) 
      k point  65:   (   0.00000  0.00000  0.25133 ) 
      k point  66:   (   0.00000  0.00000  0.25525 ) 
      k point  67:   (   0.00000  0.00000  0.25918 ) 
      k point  68:   (   0.00000  0.00000  0.26311 ) 
      k point  69:   (   0.00000  0.00000  0.26704 ) 
      k point  70:   (   0.00000  0.00000  0.27096 ) 
      k point  71:   (   0.00000  0.00000  0.27489 ) 
      k point  72:   (   0.00000  0.00000  0.27882 ) 
      k point  73:   (   0.00000  0.00000  0.28274 ) 
      k point  74:   (   0.00000  0.00000  0.28667 ) 
      k point  75:   (   0.00000  0.00000  0.29060 ) 
      k point  76:   (   0.00000  0.00000  0.29452 ) 
      k point  77:   (   0.00000  0.00000  0.29845 ) 
      k point  78:   (   0.00000  0.00000  0.30238 ) 
      k point  79:   (   0.00000  0.00000  0.30631 ) 
      k point  80:   (   0.00000  0.00000  0.31023 ) 
      k point  81:   (   0.00000  0.00000  0.31416 ) 
      k point  82:   (   0.00000  0.00000  0.31809 ) 
      k point  83:   (   0.00000  0.00000  0.32201 ) 
      k point  84:   (   0.00000  0.00000  0.32594 ) 
      k point  85:   (   0.00000  0.00000  0.32987 ) 
      k point  86:   (   0.00000  0.00000  0.33379 ) 
      k point  87:   (   0.00000  0.00000  0.33772 ) 
      k point  88:   (   0.00000  0.00000  0.34165 ) 
      k point  89:   (   0.00000  0.00000  0.34558 ) 
      k point  90:   (   0.00000  0.00000  0.34950 ) 
      k point  91:   (   0.00000  0.00000  0.35343 ) 
      k point  92:   (   0.00000  0.00000  0.35736 ) 
      k point  93:   (   0.00000  0.00000  0.36128 ) 
      k point  94:   (   0.00000  0.00000  0.36521 ) 
      k point  95:   (   0.00000  0.00000  0.36914 ) 
      k point  96:   (   0.00000  0.00000  0.37306 ) 
      k point  97:   (   0.00000  0.00000  0.37699 ) 
      k point  98:   (   0.00000  0.00000  0.38092 ) 
      k point  99:   (   0.00000  0.00000  0.38485 ) 
      k point 100:   (   0.00000  0.00000  0.39270 ) 

   <global routines>
             clock number :     2

                    bands :     0.90s CPU 

                want_init :     0.07s CPU


